src/clj/job_streamer/control_bus/component/scheduler.clj

```diff
@@ -122,11 +123,29 @@
 (defn validate-format [cron-notation]
   (CronExpression/validateExpression cron-notation))
 
+(defn hh:MM? [hh:MM-string]
+  (and (some? hh:MM-string )
+       (re-find #"^\d{2}:\d{2}$" hh:MM-string)
+       (let [hhMM (-> hh:MM-string (str/replace ":" "") Long/parseLong)
+             hh (quot  hhMM 100)
+             MM (mod hhMM 100)]
+         ;range
+         (and (<= 0 hh 23)  (<= 0 MM 59)))))
+
+(defn to-ms-from-hh:MM [hh:MM-string]
+  (if-not (hh:MM? hh:MM-string)
+    0
+    (let [hhMM (-> hh:MM-string (str/replace ":" "") Long/parseLong)
+          hh (quot  hhMM 100)
+          MM (mod hhMM 100)]
```

Comment:
It is a little redundancy. The hh and the MM can be bound at once, like following.

```
(let [[hh MM] (-> hh:MM-string (str/split #":") (#(map read-string %)))] 
```